docs/partner_editable/architecture.adoc

```diff
@@ -1,28 +1,34 @@
 Deploying this Quick Start for a new virtual private cloud (VPC) with
-default parameters builds the following _{partner-product-name}_ environment in the
+default parameters builds the following {partner-product-name} environment in the
 AWS Cloud.
 
 :xrefstyle: short
 [#architecture1]
-.Quick Start architecture for _{partner-product-name}_ on AWS
+.Quick Start architecture for {partner-product-name} on AWS
 [link=images/architecture_diagram.png]
 image::../images/architecture_diagram.png[Architecture,width=100%,height=100%]
 
+//TODO Shivansh, Do you know why Figure 1, above, gets an outline applied while the figures that really need the outline (the screen shots) do not? See Figures 3, 4, 5, 6, etc. We created the outlining standard specifically for screen shots. How do we fix this at the global level?
```

Comment:
I think its done using the CSS from the boilerplate. Is it happening only with this QS, or other QS as well?

Reply:
@sshvans I discovered what was missing. For every figure that has a screen shot, we need to manually add this line with a unique filename: `[link=images/<filename>.png]`

I've gone through the guide and added borders to all the figures.